src/python/pants/jvm/resolve/coursier_fetch.py

```diff
@@ -347,6 +418,14 @@ async def coursier_resolve_lockfile(
     if len(artifact_requirements) == 0:
         return CoursierResolvedLockfile(entries=())
 
+    # Transform requirements with local JAR files into coordinates, and yoink the relevant
+    # files into the coursier sandbox
+    better_artifacts = await Get(
```

Comment:
Maybe rename to `artifacts_with_local_files`? As a reader, "better" requires reference to a standard to evaluate by.